hc/api/tests/test_check_model.py

```diff
@@ -36,3 +43,12 @@ def test_paused_check_is_not_in_grace_period(self):
         self.assertFalse(check.in_grace_period())
 
     ### Test that when a new check is created, it is not in the grace period
+    def test_new_check_is_not_in_grace_period(self):
+        check = Check()
+
+        check.status = "up"
+        check.last_ping = timezone.now() - timedelta(days=1, minutes=30)
+        self.assertTrue(check.in_grace_period())
+
+        check.status = "paused"
```

Comment:
Why this though? Might be out of the scope of the current test. I think this was already asserted in the previous test `test_paused_check_is_not_in_grace_period`